This patch fixes the build failure on AArch64 reported
at <https://github.com/seccomp/libseccomp/pull/191>.

From cc21c1b48d35f9d34ef2da0e184af3855bfeee5f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Wed, 20 Nov 2019 14:11:12 -0500
Subject: [PATCH] tests: use openat instead of open

On arm64, __NR_open is not defined, openat is always used. Let's use openat
instead, which is defined for architectures currently.
---
 tests/15-basic-resolver.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/15-basic-resolver.c b/tests/15-basic-resolver.c
index 6badef1..4884faf 100644
--- a/tests/15-basic-resolver.c
+++ b/tests/15-basic-resolver.c
@@ -55,7 +55,7 @@ int main(int argc, char *argv[])
 	unsigned int arch;
 	char *name = NULL;
 
-	if (seccomp_syscall_resolve_name("open") != __NR_open)
+	if (seccomp_syscall_resolve_name("openat") != __NR_openat)
 		goto fail;
 	if (seccomp_syscall_resolve_name("read") != __NR_read)
 		goto fail;
